### DISCUSSION TITLE
Accessibility contrast ratio of foreground and background color must be above 4.5:1

### DISCUSSION THREAD

**Discussion by thesobercoder** [tailwindlabs/tailwindcss#2157] [General]
Came across this amazing free tool which lets you check if you the background and foreground color used in a context has a contrast more than 4.5:1.

@adamwathan This might be very useful for your consideration in Tailwind UI.

https://www.levelaccess.com/color-contrast-checker/

**Comment by adamwathan** [comment]
Nice! We use this one a lot: https://webaim.org/resources/contrastchecker/

**Comment by scottromack** [comment]
There should be a node package that checks this in development?
